src/components/CardInputs/reps.js

```diff
@@ -0,0 +1,13 @@
+export default function CardInputReps({reps, handleRepInput}) {
+	return (
+		<label>
+			Reps:{' '}
+			<input
+				type="text"
+				name="reps"
+				value={reps}
+				onChange={event => handleRepInput(event.target.value)}
+			></input>
+		</label>
```

Comment:
And you don't need this closing tag either  🧐

```suggestion
			<input
				type="text"
				name="reps"
				value={reps}
				onChange={event => handleRepInput(event.target.value)}
			 / >
```